sympy/matrices/matrices.py

```diff
@@ -1215,7 +1215,7 @@ def eigenvals(self, error_when_incomplete=True, **flags):
         =====
 
         Eigenvalues of a matrix `A` can be computed by solving a matrix
-        equation `\det(A - \lambda I) = 0`
+        equation ``\det(A - \lambda I) = 0``
```

Comment:
This and similar instances of LaTex Math, should continue to use single back-ticks.

Reply:
@sidhantnagpal updated please review.